include/ringmesh/geogram_extension/geogram_mesh.h

```diff
@@ -44,9 +44,14 @@
 
 #include <ringmesh/geogram_extension/geogram_extension.h>
 
-#include <ringmesh/mesh/mesh.h>
 #include <ringmesh/mesh/mesh_index.h>
 
+#include <ringmesh/mesh/line_mesh.h>
+#include <ringmesh/mesh/mesh_set.h>

```

Comment:
Are these includes nedded? I think you can remove mesh_set.h one and maybe forward declare others?

Reply:
I removed mesh_set.h from here and added it in 
src/ringmesh/geomodel/core/geomodel_mesh.cpp
where the class mesh_set is needed